### PR TITLE
Use new version of PeerNet with the `stream_limiter` set

### PR DESCRIPTION
**Pull request by litchipi**
Uses the changes from https://github.com/massalabs/PeerNet/pull/55 inside the `massa_protocol_{worker,exports}`

* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

File: massa-node/src/operation_injector.rs [hunk 1]
```diff
@@ -113,7 +114,7 @@ pub fn start_operation_injector(
             protocol_controller
                 .propagate_operations(storage.clone())
                 .unwrap();
-            std::thread::sleep(Duration::from_secs(1));
+            std::thread::sleep(Duration::from_secs(1).saturating_sub(now.elapsed()));
```
Comment (litchipi) [inline massa-node/src/operation_injector.rs:117]: This is a fix for the operation injector, I'll take it out of this PR and create a new one especially for this

Comment (AurelienFT) [reply massa-node/src/operation_injector.rs:117]: You can let it there it's ok

Comment (litchipi) [reply massa-node/src/operation_injector.rs]: PR: #4194

**Review by Leo-Besancon — APPROVED** [review]
Congrats!

**Review by AurelienFT — APPROVED** [review]

**Comment by litchipi** [comment]
I just fixed tests in `massa-bootstrap`. Because some `Limiter` were created in advance, they had the time to stack a bit of tokens before the read, so it took less time than it was expected.
To solve this, I added a `std::thread::sleep` to stack to the maximum the Limiters, then assert they take longer than expected (but lower the expectation as we stacked up)

**Pull request merged**